source/_posts/2019-10-07-upgrading-psr-interfaces.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Upgrading PSR interface'
+description: >-
+Greetings, PHP. The PHP Framework Interoperability Group is **actively looking for feedback on a way to safely update and modernize several existing PSRs**. We think this is a workable approach, but before embarking on it we want to get feedback from the broader community. That means you.
+date: '2019-10-07T14:11:13.604Z'
+categories: []
+keywords: []
+slug: upgrading-psr-interfaces
+tags:
+ - php-fig
+ - bylaws
+author: crell
+layout: post
+use:
+ - authors
+ - posts
+---
+
+## PHP-FIG wants your help to modernize PSRs!
+
+Greetings, PHP. The PHP Framework Interoperability Group is **actively looking for feedback on a way to safely update and modernize several existing PSRs**. We think this is a workable approach, but before embarking on it we want to get feedback from the broader community. That means you.
+
+### The problem
+
+PHP-FIG has been publishing PHP interface specifications for years now, from the original [PSR-3 logger interface](https://github.com/php-fig/log) in 2016 to the latest, the [PSR-14 Event Dispatcher interfaces](https://github.com/php-fig/event-dispatcher) this last spring. All of them are, naturally, a product of their time. That is, **they were written for the version of PHP that was current** at the time they were released.
```

Comment:
PSR-3 was long before 2016.  Like, 2011 or something I think.

Reply:
Yes, you're right. I thought it too, but the GitHub release confused me: https://github.com/php-fig/log/releases/tag/1.0.0

The commit is in fact from December 2012.